src/device-registry/utils/create-monitor.js

```diff
@@ -18,7 +18,6 @@ const logger = log4js.getLogger(
 
 const qs = require("qs");
 const QRCode = require("qrcode");
-const { kafkaProducer } = require("../config/kafkajs");
```

Comment:
Thanks @BenjaminSsempala , could you add some guiding PR review comments about this deletion?